criu/files-reg.c

```diff
@@ -155,15 +155,14 @@ static int trim_last_parent(char *path)
 static int copy_chunk_from_file(int fd, int img, off_t off, size_t len)
 {
 	char *buf = NULL;
-	int ret;
-
```

Comment:
Please don't remove newline between variable declaration and next block of code.

Reply:
@Snorch I have made these changes. Please let me know if there any other changes to be made.